README.md

```diff
@@ -124,24 +117,29 @@ Install with
 sudo make install
 ```
 
-Building from git
------------------
-
-You should have python2.7+ installed.
+## Building from git
 
 Download project with
 
 		git clone --recursive https://github.com/rockdaboot/libpsl
 		cd libpsl
 
-Build with GNU autotools
+### Building with GNU autotools
+
+Prerequisites:
+  - python2.7+ (for converting the PSL list)
+  - basic C development tools (compiler, linker)
+  - git, make, autoconf, autoconf-archive, autopoint, automake, autotools,
+  - libtool, gettext, m4, pkg-config
```

Comment:
A lot of this can be summarized as "need autotools". Mentioning autopoint and m4 in particular are very specific details that are really abstracted away by the primary projects they are bundled with or a dependency of -- m4 is required by autoconf since autoconf processes m4 files, autopoint is part of gettext.

Reply:
Indeed, `m4` has been pulled in as dependency here on Debian testing, but not so for `autopoint` nor `autotools-dev`. So I'd rather mention these tools/packages.

Removing `git` because without it you won't be able to fetch the sources!? (Mostly)

Reply:
I did not realize autopoint was a separate package in Debian. How utterly pointless...

Reply:
It has been part of gettext a while ago, AFAIR. There must be some reason to separate both, but idea tbh.